db/views/reporting_daily_follow_ups_v01.sql

```diff
@@ -0,0 +1,94 @@
+WITH follow_up_blood_pressures AS (
+  SELECT DISTINCT ON (patient_id, facility_id, day_of_year)
+    p.id AS patient_id,
+    p.gender::gender_enum as patient_gender,
+    bp.id as visit_id,
+    'BloodPressure' AS visit_type,
+    bp.facility_id,
+    bp.user_id,
+    bp.recorded_at AS visited_at,
+    cast(EXTRACT(DOY FROM bp.recorded_at AT TIME ZONE 'UTC' at time zone (SELECT current_setting('TIMEZONE'))) as integer) AS day_of_year
+ FROM patients p
+    INNER JOIN blood_pressures bp
```

Comment:
Note that we have to process against the real transactional tables here with the hopes that this will be fast enough to run on its own at least a few times a day.  This will allow us to display update daily counts throughout the day in the progress tab.